acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/policy/admin/PolicyCreationStepsDefinitions.java

```diff
@@ -46,6 +46,7 @@
  *
  * @author 212338046
  */
+//CHECKSTYLE:OFF
```

Comment:
Checkstyle is off because these cucumber tests don't follow the naming conventions in checkstyle.